README.md

```diff
@@ -96,7 +96,7 @@ Just like with setup options, you can edit these in game. For example, you could
 
 This will automatically save to the chataliases config file.
 
-By default the plugin uses the ``cfg/sourcemod/pugsetup/live.cfg`` config when going live. You are free to change this file all you want (or change which file is used via the ``sm_pugsetup_live_cfg`` cvar). (Note: if you use knife rounds, make sure ``mp_give_player_c4 1`` is in this file!)
+By default the plugin uses the ``cfg/sourcemod/pugsetup/live.cfg`` config when going live. You are free to change this file all you want (or change which file is used via the ``sm_pugsetup_live_cfg`` cvar). (Note: if you use knife rounds, and the knife config is executed, the cvars it sets are saved before execution then restored when the knife round finishes.)
```

Comment:
This statement doesn't really make sense:

"if you use knife rounds, and the knife config is executed"

If you use knife rounds the knife config is always executed.

I'd change it to something like:

 (Note: if you use knife rounds, the knife config will be executed after the live config, but all the cvars it changed will be restored to their previous values after the knife round.)